python/json_util.py

```
@@ -4,6 +4,55 @@
+            "type": "node",
+            "id": value.id,
+            "labels": [label.name for label in value.labels],
+            "properties": {k: _convert_value_to_json_compatible(v) for k, v in value.properties.items()}
+        }
+    elif isinstance(value, mgp.Edge):
+        return {
+            "type": "relationship",
+            "id": value.id,
+            "start": value.from_vertex.id,
+            "end": value.to_vertex.id,
+            "type": value.type.name,
```

Ruff (0.11.9)

Dictionary key literal `"type"` repeated

(F601)